docs/scos/dev/feature-walkthroughs/201811.0/gift-cards-feature-walkthrough.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gift Cards feature walkthrough
+last_updated: Sep 28, 2021
+description: The Gift Cards feature enables you to create a virtual product (a gift card) with a chosen value amount
+template: concept-topic-template
+---
+
+The _Gift Cards_ feature enables you to create a virtual product (a gift card) with a chosen value amount. The purchase of a gift card generates an individual code that is used as a payment method during checkout.
```

Comment:
Technically, all the products are virtual. I suggest using "special product type".